### PR TITLE
Update default icon

### PR DESCRIPTION
**Pull request by atjn**
Hello and thanks for making this very cool project!

I noticed that most of your design has moved to a new flat Friendica icon, but that the project still uses the old icon in certain places. I thought I would help update that.

My steps:

### Update `.jpg` icons to `.png` everywhere
There is no reason to use JPEG images when PNG provides better compression.

### Update `.png` icons to `.svg` where possible
Ideally, PNG images should also not be used because the single SVG image can serve all the different PNG sizes while using much less bandwidth.
Unfortunately, changing an image to `.svg` sometimes requires changes to the rendering engine and/or page layout, so I was not able to change this everywhere because I don't know how the image is rendered (for example when publishing to Mastodon).

### Update SVG design
Update the SVG design to the new flat logo.

### Only update design for strictly necessary PNGs
As mentioned above, we should ideally only be using the SVG for everything, so to make it clear that the other image types are not to be used, I did not update their designs. The only exception is for the PNGs that are still used because I wasn't sure if I could replace them.

This PR is related to #11452

### CODE REVIEW

File: doc/BBCode.md [hunk 1]
```diff
@@ -65,17 +65,17 @@ table.bbcodes > * > tr > th {
   <td><a href="http://friendi.ca" target="external-link">Friendica</a></td>
 </tr>
 <tr>
-  <td>[img]https://raw.githubusercontent.com/friendica/friendica/stable/images/friendica-32.jpg[/img]</td>
-  <td><img src="https://raw.githubusercontent.com/friendica/friendica/stable/images/friendica-32.jpg" alt="Immagine/foto"></td>
+  <td>[img=32x32]https://raw.githubusercontent.com/friendica/friendica/stable/images/friendica.svg[/img]</td>
+  <td><img src="https://raw.githubusercontent.com/friendica/friendica/stable/images/friendica.svg" style="width: 32px;" alt="Immagine/foto"></td>
```
Comment (MrPetovan) [inline doc/BBCode.md]: This breaks the documentation steps for using the `[img]` tag:
- Without any parameters
- With an alt text
- With size parameter
```suggestion
  <td>[img]https://raw.githubusercontent.com/friendica/friendica/stable/images/friendica-32.png[/img]</td>
  <td><img src="https://raw.githubusercontent.com/friendica/friendica/stable/images/friendica-32.png"></td>
```

File: doc/de/BBCode.md [hunk 2]
```diff
@@ -65,17 +65,17 @@ table.bbcodes > * > tr > th {
   <td><a href="http://friendi.ca" target="external-link">Friendica</a></td>
 </tr>
 <tr>
-  <td>[img]https://raw.githubusercontent.com/friendica/friendica/stable/images/friendica-32.jpg[/img]</td>
-  <td><img src="https://raw.githubusercontent.com/friendica/friendica/stable/images/friendica-32.jpg" alt="Immagine/foto"></td>
+  <td>[img=32x32]https://raw.githubusercontent.com/friendica/friendica/stable/images/friendica.svg[/img]</td>
+  <td><img src="https://raw.githubusercontent.com/friendica/friendica/stable/images/friendica.svg" style="width: 32px;" alt="Immagine/foto"></td>
```
Comment (MrPetovan) [inline doc/de/BBCode.md]: This breaks the documentation steps for using the `[img]` tag:
- Without any parameters
- With an alt text
- With size parameter
```suggestion
  <td>[img]https://raw.githubusercontent.com/friendica/friendica/stable/images/friendica-32.png[/img]</td>
  <td><img src="https://raw.githubusercontent.com/friendica/friendica/stable/images/friendica-32.png"></td>
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]
Thank you for your work! I see you've significantly changed the Friendica logo when generating the PNG and SVG versions. I'd rather we keep the black border, otherwise we're losing a fair bit of contrast. I don't mind the flattening of the image (removing the light reflection at the top and the shadow at the bottom) but I believe the black outline outside and inside should stay.

**Comment by atjn** [comment]
> I see you've significantly changed the Friendica logo when generating the PNG and SVG versions. I'd rather we keep the black border, otherwise we're losing a fair bit of contrast.

I have simply used the icon that you have started using everywhere else:

image
image
image
image
image

This is of course up to you, but I personally think it looks odd to specifically give this logo a border when all of the others use a borderless design. (and I don't mind the smaller contrast)

File: doc/BBCode.md (hunk 1 above)
Comment (atjn) [reply doc/BBCode.md]: It was in an effort to remove as many PNG references as possible, but I see the issue.

File: doc/de/BBCode.md (hunk 2 above)
Comment (atjn) [reply doc/de/BBCode.md]: It was in an effort to remove as many PNG references as possible, but I see the issue.

**Comment by MrPetovan** [comment]
I see what you mean, but I don't feel confortable approving on my own such a significant change of the project's logo. I'll request reviews from other members.

**Review by MrPetovan — APPROVED** [review]

**Review by fabrixxm — APPROVED** [review]

**Review by annando — APPROVED** [review]

**Comment by tobiasd** [comment]
I never really liked the flattening of the icons - but that's a general issue and not only for Friendica 😉

**Review by tobiasd — APPROVED** [review]

**Comment by MrPetovan** [comment]
Thank you for your reviews, I believe we can move forward with this PR!

**Pull request merged**